mrp_multi_level/readme/HISTORY.rst

```diff
@@ -1,3 +1,13 @@
+13.0.1.1.0 (2020-02-20)
```

Comment:
13.0.1.2.0

Once you will be changing this, would you please add the proper 13.0.1.1.0 section (description can be just the multicompany fixes done lately https://github.com/OCA/manufacture/commits/13.0/mrp_multi_level)?

Reply:
Right! Thanks